mapflow/mapflow.py

```diff
@@ -3140,6 +3179,40 @@ def update_processing(self):
         dialog.setup(processing)
         dialog.deleteLater()
 
+    def check_project_sharing(self):
+        if len(self.dlg.projectsCombo) != 0:
+            self.current_project = self.projects[self.dlg.projectsCombo.currentIndex()]
+            project_name = self.current_project.name
+            if self.current_project.shareProject:
+                # Get user role, if project is shared
+                users = self.current_project.shareProject.users
+                self.user_role = UserRole.owner
+                if users:
```

Comment:
it seems that "if users" here is redundant? If None breaks the "for user in users", we can ensure that None translates to [] on inittialization of ShareProject